client/my-sites/post-share/connections-list.jsx

```diff
@@ -107,6 +64,40 @@ class ConnectionsList extends PureComponent {
 	}
 }
 
+export const ConnectionsWarning = ( {
```

Comment:
it's in the same `connections-list.jsx` file. The idea was trying to keep the whole connections list stuff in the same one, but maybe we should move this `<Notice />` out of `<ConnectionsList /> because in honor of the truth, although they are related, they are different things.